third_party/rust/repos.bzl

```diff
@@ -13,14 +14,16 @@ def rust_repos():
     )
 
     # Boring is only used to build Mundane.
-    http_archive(
+    dynamic_archive(
```

Comment:
Are these the only 2 repos we have that require this different approach to verification?

Reply:
Yes, right now they are.